tealer/utils/instruction_utils.py

```diff
@@ -0,0 +1,77 @@
+from typing import List, Dict
+
+
+def instruction_cost(ins_str: str, version: int) -> int:
```

Comment:
Maybe this should take an Instruction and not a string, so that we don't need to worry about formating when we call it?

Reply:
Yes, I will update the function accordingly 👍